indexer/api/api.go

```diff
@@ -1,102 +1,36 @@
 package api
 
 import (
-	"encoding/json"
+	"fmt"
 	"net/http"
 
+	"github.com/ethereum-optimism/optimism/indexer/api/handlers"
```

Comment:
might rename this routes in a future pr yesterday I liked handlers today I like routes we'll see what I like when this is done and in the clean up phase